### PR TITLE
fix(autorenew-tracking): track events in autorenew

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

add tracking for autorenew button and modal submit

### Benefits

Helps track more events

### Possible Drawbacks

None

### Applicable Issues

resolves MBE-248

### CODE REVIEW

File: client/app/account/billing/payment/method/billing-payment-method.html [hunk 1]
```diff
@@ -118,10 +118,10 @@ <h2 data-translate="billing_payment_method_title"></h2>
             </oui-action-menu>
 
             <extra-top>
-                <a class="oui-button oui-button_secondary"
-                   data-ui-sref="app.account.billing.payment.method.add"
-                   data-translate="billing_payment_method_table_top_add_method">
-                </a>
+                <oui-button data-variant="secondary"
+                   data-on-click="$ctrl.goToAddPayment()"
+                   data-text="{{ :: 'billing_payment_method_table_top_add_method' | translate }}">
```
Comment (antleblanc) [inline client/app/account/billing/payment/method/billing-payment-method.html]: Prefer adding a `<span data-translate="billing_payment_method_table_top_add_method"></span>` inside the `<oui-button>` element instead.

Beause the `text` attribute is deprecated.

ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/button#deprecated

Thanks!

File: client/app/account/billing/payment/method/billing-payment-method.controller.js [hunk 2]
```diff
@@ -41,6 +43,20 @@ export default class BillingPaymentMethodCtrl {
...
+   */
+  goToAddPayment() {
+    this.atInternet.trackClick({
+      name: 'add_payment_types',
+      type: 'action',
+      chapter1: 'payment_types',
+      chapter2: 'payment_types',
+      chapter3: 'add',
+    });
+    this.$state.go('app.account.billing.payment.method.add');
```
Comment (FredericEspiau) [inline client/app/account/billing/payment/method/billing-payment-method.controller.js]:
```suggestion
    return this.$state.go('app.account.billing.payment.method.add');
```

File: client/app/account/billing/payment/method/billing-payment-method.controller.js [hunk 3]
```diff
@@ -41,6 +43,20 @@ export default class BillingPaymentMethodCtrl {
   =            EVENTS            =
   ============================== */
 
+  /**
+   *  Navigate to add payment method
+   */
+  goToAddPayment() {
+    this.atInternet.trackClick({
```
Comment (FredericEspiau) [inline client/app/account/billing/payment/method/billing-payment-method.controller.js]: Doesn't this return a `Promise` ?

Comment (varun257) [reply client/app/account/billing/payment/method/billing-payment-method.controller.js]: No it doesnt.
https://github.com/ovh-ux/ng-at-internet/blob/master/src/provider.js#L345

**Comment by jleveugle** [comment]
Hello @varun257,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**